Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -188,13 +188,8 @@ class ActivityStreamTest: BaseTestCase {
         waitForExistence(app.cells.staticTexts.element(boundBy: 0), timeout: 10)
 
         navigator.nowAt(TabTray)
-        if iPad() {
-            waitForExistence(app.collectionViews.element(boundBy: 1).cells.staticTexts["Wikipedia"], timeout: TIMEOUT)
-            app/*@START_MENU_TOKEN@*/.otherElements["Tabs Tray"].collectionViews/*[[".otherElements[\"Tabs Tray\"].collectionViews",".collectionViews"],[[[-1,1],[-1,0]]],[1]]@END_MENU_TOKEN@*/.cells["Wikipedia"].tap()
```

Comment:
Why are the comments such as `@START_MENU_TOKEN@` and `@END_MENU_TOKEN@` are present in the code previously? They are not visible from XCode but are visible from a text editor.